monte_carlo.py

```diff
@@ -24,22 +24,49 @@
 ENERGY_HEATING_MWH = 866
 ENERGY_COOLING_MWH = 912
 # FIXME: you have 40 in readme, not 4 
+# We need both. 40 is the COP of the groundwater pump, 4 of the heat pump.
 COP_DISTRIBUTION_PARAM = dict(left=4, mode=4.5, right=5)
 
-def seed_cop():
+# What is missnig is the COP of the groundwater pump
+COP_GW_PUMP_DISTRIBUTION_PARAM = dict(left=35, mode=40, right=45)
+
+# I changed this to a triangular distributon. 
+PRICE_DISTRIBUTION_PARAM = dict(left=140, mode=150, right=160)
+
+def seed_cop_hp():
     return triangular(**COP_DISTRIBUTION_PARAM)
 
 def seed_price():
-    return normal(0.15, 0.05) 
+    return triangular(**PRICE_DISTRIBUTION_PARAM) 
+
+def seed_cop_gw_pump():
+    return triangular(**COP_GW_PUMP_DISTRIBUTION_PARAM) 
 
 # QUESTION: is this right for a groundwater cycle? choose one cop for both seasons?
+# Yes, beacause we are using the same pump to extract the groundwater during winter- & summertime
+
 def gw_consumption():
-    return (ENERGY_HEATING_MWH + ENERGY_COOLING_MWH) / seed_cop() 
+    return (ENERGY_HEATING_MWH + ENERGY_COOLING_MWH) / seed_cop_gw_pump() 
 
 def gw_cost():
     """Groundwater cycle cost, euro"""
     return gw_consumption() * seed_price()
 
+# Calculations of the elec. costs of the heat pump
+def hp_output():
+    return (ENERGY_HEATING_MWH * seed_cop_hp()) / (seed_cop_hp() - 1)
+
```

Comment:
It would have helped to be able to follow what is energy in and energy out of the heatpump. Just some words about what gets in and out.